README-sandbox.md

```diff
@@ -59,15 +59,15 @@ $ chmod -R 0700 keys
 Don't forget to set key files' permissions to `0700` or else the
 client will refuse to import them.
 
-Additionally, the block producer's key should be copied to the `wallet`
+Additionally, the block producer's key should be copied to the `wallets`
 directory in the node's config dir. This directory doesn't exist yet
 probably, so one can create it by hand or try launching the node to
 set up the config directory. The filename should be identical to the
 block producer's public key.
 
 ```shell
-$ mkdir -p .mina-config/wallet/store
-$ cp keys/block-producer.key .mina-config/wallet/store/<public-key>
+$ mkdir -p .mina-config/wallets/store
+$ cp keys/block-producer.key .mina-config/wallets/store/$(cat keys/block-producer.key.pub)
```

Comment:
`$(cat keys/block-producer.key.pub)` is not really needed, but I found it convenient and meaningful.